pages/news/2020-XX-XX-concrete-action.html

```diff
@@ -0,0 +1,63 @@
+title: Concrete Action: Taking a Stand with Terminology and Recruitment
+author: Mixxx, April
+date: 2020-06-08 01:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown:"extra" %}
+
+# &#35;BlackLivesMatter
+
+Alongside the protesters in the US and around the world, the Mixxx team unequivocally renounces police brutality.
+The systemic racism that burdens people of color every day is the responsibility of white people around the world to address.
+Mixxx' core development team feels it necessary not to remain silent on this issue, for "silence in the face of injustice is complicity with the oppressor."
+
+Going beyond the hashtag, we have been discussing concrete actions we can take as a community and today we announce two efforts: first, to **eliminate "master" and "slave" terminology from our code base** and second, to **recruit paid interns from underrepresented communities** to join our contributor base.
```

Comment:
Slave has been eliminated already, and we need to give a reason ..

"first, to eliminate "master" where it can be associated with the master-slave pattern and remains to the unspeakable history of 400 years of systematic violence against ..."

Reply:
This paragraph is meant to be a brief summary introduction to announce the specific actions.  I'd prefer not to clutter it up with the full explanation here.  I also think it's not necessary for us to explain the reason for eliminating the terms.  Maybe we can link somewhere below? "for a background on this movement..."

Reply:
> first, to **eliminate "master" and "slave" terminology

This is not correct, because "slave" is already eliminated. 

It is very important to explain the reason why we do it because it is not obvious in the international context. 

I have learned the master-slave pattern during my networking classes and because of my limited shingle skills I took the term slave as "Secondary Network Device". A lot later I realized that this term also means "thrall". 

Some days ago a college uses the words master and slave in the network context and I told him not to do that. 
He also hadn't not realized that this term is taken from the slavery. So I think this is at least in Germany a common fail.

Reply:
I'll add a couple more links here and below -- if people are unfamiliar with the context they can read much more full discussons on their own time.  I really don't want to clog up this concise initial statement of intent with a full history.   (Note that the later links for the major projects also contain lots of discussion about the background of the movement, so if anyone is confused about what we're talking about there's lots to read).